internal/consumer/push_consumer.go

```diff
@@ -99,9 +104,37 @@ func NewPushConsumer(consumerGroup string, opt primitive.ConsumerOption) (PushCo
 	} else {
 		p.submitToConsume = p.consumeMessageCurrently
 	}
+
+	ChainInterceptor(p)
+
 	return p, nil
 }
 
+// ChainInterceptor chain list of interceptor as one interceptor
+func ChainInterceptor(p *pushConsumer) {
+	interceptors := p.option.Interceptors
+	switch len(interceptors) {
+	case 0:
+		p.interceptor = nil
+	case 1:
+		p.interceptor = interceptors[0]
+	default:
+		p.interceptor = func(ctx context.Context, req, reply interface{}, invoker primitive.CInvoker) error {
```

Comment:
use iteration, no recursive

Reply:
iter can not determine continue or not，recursive is better if user do want to continue to execute

Reply:
i mean like this: 
```
interceptor := = interceptors[0]
for i := 1 ; i < len(interceptors); i++ {
  interceptor = func(ctx context.Context, req, reply interface{}, invoker primitive.CInvoker) error {
   return interceptors[i](ctx, req, reply, interceptor)
  }
}
```
so we cloud remove `getChainedInterceptor`

Reply:
ok